nekoyume/Assets/_Scripts/UI/Widget/ArenaBattlePreparation.cs

```diff
@@ -219,6 +204,52 @@ public override void Close(bool ignoreCloseAnimation = false)
 
         #endregion
 
+        private void UpdateAvatarState()
+        {
+            var avatarState = Game.Game.instance.States.CurrentAvatarState;
+            var arenaAvatarState = RxProps.PlayersArenaParticipant.Value.AvatarState;
+            var currentBlockIndex = Game.Game.instance.Agent.BlockIndex;
+
+            var arenaItems = arenaAvatarState.inventory.Items;
+            for (int i = arenaItems.Count - 1; i > 0; i--)
```

Comment:
```
for (int i = arenaItems.Count - 1; i > 0; i--)
```
This `for` loop seems not intuitive. Did you mean it?

Reply:
There is a possibility the element of `arenaAvatarState.inventory` will be modified in the loop.
```
line.237
if (existItem is null)
{
    arenaAvatarState.inventory.RemoveItem(arenaItems[i]);
}
```